gulp/karma.js

```diff
@@ -111,5 +33,7 @@ module.exports = (gulp, plugins, blueprint) => {
         });
     });
 
-    gulp.task("karma", (done) => rs(...blueprint.taskMapper("karma", "karma-"), done));
+    // running in sequence so output is human-friendly
+    // (in parallel, all suites get interspersed and it's a mess)
```

Comment:
for another PR: try running expensive parallel tasks in separate CircleCI containers